### PR TITLE
V15 Standalone runtime

### PR DESCRIPTION
**Pull request by joshuahuburn**
#### Description
Implements the new runtime starting code required for v15 standalone mode.
Does not implement launch config generation or spatial output. These come in later PRs.

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialWorkerConnectionTest.cpp [hunk 1]
```diff
@@ -138,8 +138,11 @@ bool FSendReserveEntityIdsRequest::Update()
 {
 	uint32_t NumOfEntities = 1;
 	USpatialWorkerConnection* Connection = ConnectionManager->GetWorkerConnection();
-	Connection->SendReserveEntityIdsRequest(NumOfEntities);
-	Connection->Flush();
+	if (Connection)
```
Comment (jessicafalk) [inline SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialWorkerConnectionTest.cpp:141]: could connection be a nullptr here now?

File: SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp [hunk 2]
```diff
@@ -401,365 +242,97 @@ void FLocalDeploymentManager::TryStartLocalDeployment(FString LaunchConfig, FStr
...
-		UE_LOG(LogSpatialDeploymentManager, Error, TEXT("Json parsing of spot list result failed. Result: %s"), *SpotListResult);
-	}
-
-	const TSharedPtr<FJsonObject>* SpotJsonContent = nullptr;
-	if (bParsingSuccess && SpotJsonResult->TryGetObjectField(TEXT("content"), SpotJsonContent))
-	{
-		const TArray<TSharedPtr<FJsonValue>>* JsonDeployments;
-		if (!SpotJsonContent->Get()->TryGetArrayField(TEXT("deployments"), JsonDeployments))
+		// If the runtime did not stop after some timeout then inform the user as something is amiss.
+		if (RuntimeProcess->GetDuration().GetTotalSeconds() > RuntimeStopTime + 5.0)
```
Comment (jessicafalk) [inline SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp:317]: no magic number :) let's make it a field

File: SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialWorkerConnectionTest.cpp (hunk 1 above)
Comment (joshuahuburn) [reply SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialWorkerConnectionTest.cpp:141]: It always could be. I had to fix these as these connection tests cause a reading from invalid memory exception which crashes the editor, if the first test to setup the connection fails :o

File: SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp (hunk 2 above)
Comment (joshuahuburn) [reply SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp:317]: Done

**Review by jessicafalk — APPROVED** [review]

**Review by mimistiv — APPROVED** [review]
nice

**Review by ilievailiyana — APPROVED** [review]

**Pull request merged**